charts/poktroll-sequencer/templates/deployment.yaml

```diff
@@ -81,9 +81,7 @@ spec:
           command: ["/bin/sh", "-c"]
           args: 
             - |
-              export DA_BLOCK_HEIGHT=$(cat /root/.pocket/data/da-block-height)
-              export CELESTIA_NAMESPACE=$NAMESPACE
-              exec  poktrolld start --home=/root/.pocket/ --rollkit.aggregator=true --rollkit.da_address="{{ .Values.dataAvailability.celestia.da_rpc_url }}" --rollkit.da_start_height=$DA_BLOCK_HEIGHT --rpc.laddr=tcp://0.0.0.0:36657 --p2p.laddr="0.0.0.0:36656" --grpc.address=0.0.0.0:36658
+              exec poktrolld start --home=/root/.pocket/ --rollkit.aggregator=true --rollkit.da_address="{{ .Values.dataAvailability.celestia.da_rpc_url }}" --rollkit.da_start_height=1 --rpc.laddr=tcp://0.0.0.0:36657 --p2p.laddr="0.0.0.0:36656" --grpc.address=0.0.0.0:36658
```

Comment:
Why is the start height 1?

Reply:
@Olshansk I just wanted to remove the unnecessary complexity for the chart we use for development. There isn't really any difference for local development.